### PR TITLE
Added new translations and removed special treatment for French

### PR DESCRIPTION
**Pull request by yutamago**
Changes made:
- added Korean, Spanish and Italian translations
- corrected some errors in German translations
- removed special treatment for French (including converting translations to UTF-8)
- added missing languages to DisplayPokenameExample

### CODE REVIEW

File: library/src/main/java/com/pokegoapi/util/PokeNames.java [hunk 1]
```diff
@@ -34,12 +34,6 @@
 	public static String getDisplayName(int pokedexNr, Locale locale) {
 		ResourceBundle names = ResourceBundle.getBundle("pokemon_names", locale);
 		String str = names.getString(String.valueOf(pokedexNr));
-		if (locale == Locale.FRENCH)
-			try {
-				return new String(str.getBytes("ISO-8859-1"), "UTF-8");
-			} catch (UnsupportedEncodingException e) {
-				e.printStackTrace();
-			}
 		return str;
```
Comment (FabianTerhorst) [inline library/src/main/java/com/pokegoapi/util/PokeNames.java]: you can simplify the return now with return names.getString(String.valueOf(pokedexNr));

**Comment by yutamago** [comment]
Finally I got the identation in my last commit right. Intellij kept replacing it with spaces.

**Comment by FabianTerhorst** [comment]
:+1:

**Comment by FabianTerhorst** [comment]
When you don´t want to add more to this pr, i would merge it.

**Pull request merged**